build.sbt

```diff
@@ -55,120 +82,168 @@ val settings = Seq(
         "-Xlint:unsound-match",
         "-Xlint:nullary-override"
       )
-    ),
-  Compile / console / scalacOptions --= Seq("-Ywarn-unused:imports", "-Xfatal-warnings")
+  ),
+  Compile / console / scalacOptions --= Seq("-Ywarn-unused:imports", "-Xfatal-warnings"),
+  testFrameworks += new TestFramework("utest.runner.Framework")
 )
 
 val dependencies = Seq(
   libraryDependencies ++= Seq(
     "org.scala-lang.modules" %%% "scala-collection-compat" % "2.8.0",
     "org.scala-lang" % "scala-reflect" % scalaVersion.value % "provided",
     "com.lihaoyi" %%% "utest" % "0.8.0" % "test"
+  ),
+  addCompilerPlugin("org.typelevel" % "kind-projector" % "0.13.2" cross CrossVersion.full)
+)
+
+val publishSettings = Seq(
+  organization := "io.scalaland",
+  homepage := Some(url("https://scalaland.io")),
+  licenses := Seq("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
+  scmInfo := Some(
+    ScmInfo(url("https://github.com/scalalandio/chimney"), "scm:git:git@github.com:scalalandio/chimney.git")
+  ),
+  publishTo := sonatypePublishToBundle.value,
+  publishMavenStyle := true,
+  Test / publishArtifact := false,
+  pomIncludeRepository := { _ =>
+    false
+  },
+  pomExtra := (
+    <developers>
+      <developer>
+        <id>krzemin</id>
+        <name>Piotr Krzemiński</name>
+        <url>http://github.com/krzemin</url>
+      </developer>
+      <developer>
+        <id>MateuszKubuszok</id>
+        <name>Mateusz Kubuszok</name>
+        <url>http://github.com/MateuszKubuszok</url>
+      </developer>
+    </developers>
   )
 )
 
+val noPublishSettings =
+  Seq(publish / skip := true, publishArtifact := false)
+
+val ciCommand = (scalaSuffix: String) =>
+  Iterable(
+    "clean",
+    "scalafmtCheck",
+    "Test/scalafmtCheck",
+    s"chimneyCats$scalaSuffix/compile",
+    s"chimneyCatsJS$scalaSuffix/compile",
+    s"chimneyCatsNative$scalaSuffix/compile",
+    "coverage",
+    s"chimney$scalaSuffix/test",
+    s"chimneyCats$scalaSuffix/test",
+    s"chimney$scalaSuffix/coverageReport",
```

Comment:
Have you checked if the report contains cats coverage data on Codecov? Worst case, the task must be separately run for `chimneyCats...`

Reply:
After my last fix https://app.codecov.io/gh/scalalandio/chimney/pull/241 shows decrease in Cats coverage, so I guess it works. I would address it in another PR ideally.